index.html

```diff
@@ -1,9 +1,113 @@
-NPM Clone Nifty Penguin Magic npm Enterprise Products Solutions Resources Docs Support Search Join Log In Build amazing
-things Essential JavaScript development tools that help you go to market faster and build powerful applications using
-modern open source code. See plans Join for free Bring the best of open source to your company npm is the tool used by
-over 11,000,000 JavaScript developers around the world. Your developers already use it. Your company depends on it.
-Create an Org and get more out of the tools your team already knows and loves. Zero configuration Create an org, add
-your team, and start collaborating. Nothing to configure, set up, or manage. Team management Control who has access to
-what modules within your team namespace using straightforward team management capabilities. Familiar features npm Orgs
-has 100% parity with all the public npm registry features your developers already use. npm audit Enjoy the security
-auditing features built into the npm client, a zero-friction way to make open source software safer. Create an Org
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>npm clone</title>
+    <link href="styles.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <header>
+      <div>
+        <div>
+          <img class="blackHeart" src="images/black-heart.png" />
+          <span>NPM Clone Nifty Penguin Magic</span>
+        </div>
+        <nav class="nav-bar">
+          <ul>
+            <li><a href="">npm Enterprise</a></li>
+            <li><a href="">Products</a></li>
+            <li><a href="">Solutions</a></li>
+            <li><a href="">Resources</a></li>
+            <li><a href="">Docs</a></li>
+            <li><a href="">Support</a></li>
+          </ul>
+        </nav>
+      </div>
+
+      <div>
+        <img class="npm-logo" src="images/npm-logo.png" />
+        <div>
+          <img class="blackHeart" src="images/magnifying-glass.png" />
+          <input class="search" type="search" placeholder="Search packages" />
+        </div>
+        <button type="submit">Search</button>
+        <a href="">Join</a>
```

Comment:
Aqui tenemos dos opciones, puedes modificar con css el estilo de esta tag <a></a> o puedes agregar otra etiqueta buttom dentro de esta misma 
```suggestion
        <a href=""><button>Join</button></a>
```